webrtc.html

```diff
@@ -1012,14 +1012,30 @@ <h4>Constructor</h4>
           </li>
           <li>
             <p>If the <code>certificates</code> value in
-            <var>configuration</var> is non-empty, check that
-            the <code>expires</code> on each value is in the future. If a
-            certificate has expired or a the [[\Origin]] internal slot of
-            the certificate does not match the current origin, <a>throw</a> an
-            <code>InvalidAccessError</code>; otherwise, store the certificates.
-            If no <code>certificates</code> value was specified, one or more
-            new <code>RTCCertificate</code> instances are generated for use
-            with this <code>RTCPeerConnection</code> instance. This MAY happen
+            <var>configuration</var> is non-empty, run the following steps for
+            each certificate in certificates:
+            <ol>
+              <li>
+                <p>If the value of the certificate <code>expires</code>
+                attribute is not in the future, <a>throw</a> an
```

Comment:
specify that the "future" is "larger than performance.timeOrigin + performance.now()"

Reply:
I'm not so worried about int vs double - what I want to make clear is that nobody expects the browser to somehow magically invent yet another concept of "now" different from the ones it's already using (for instance, performance.now() is a monotonic clock, while performance.timeOrigin is tied to real-world clock, so the sum will drift a little bit away from "true" over time; by using the reference, we're saying that that's OK.) I have the same lack of concern with precisely when we check as long as it's within the current run of the main loop.

Reply:
Agreed it should hopefully not matter much.
I was hoping to find some notion of timing related to HTTP cache or certificates, which seem closer to me than performance timing. But performance timing will do the trick too